ckanext/datagovuk/controllers/user.py

```diff
@@ -18,6 +19,9 @@
 ValidationError = logic.ValidationError
 UsernamePasswordError = logic.UsernamePasswordError
 
+ckan.controllers.user.UserController.original_new = ckan.controllers.user.UserController.new
+ckan.controllers.user.UserController.new = UserController.new
```

Comment:
Why is the monkeypatching necessary? This is really confusing

Reply:
The rest of our controller overrides simply re-declare the url mapping in `plugin.py`.

Reply:
Changed the way it's done now to do a super on the original function